public/react-chat/app.jsx

```diff
@@ -1,5 +1,5 @@
 // A placeholder image if the user does not have one
-const PLACEHOLDER = '/placeholder.png';
+const PLACEHOLDER = 'https://placeimg.com/60/60/people';
```

Comment:
Also, do you want to remove the image from the repo? May as well, if we aren't using it.